src/commands/application/logs.rs

```diff
@@ -168,10 +169,25 @@ pub async fn handle_logs(
             )?;
             let resource_names = vec![format!("projects/{}", cluster.google_project_id)];
             application_progress_bar.finish_and_clear();
-
+            
             trace!("filter: {}", filter);
             trace!("resource_names: {:?}", resource_names);
 
+            // url mode only return the url
+            if *url_mode {
+                let url = url::Url::parse(&format!(
+                    "https://console.cloud.google.com/logs/query;query={}",
+                    filter
+                ))
+                .unwrap();
+                eprintln!(
+                    "URL: {}?project={}",
```

Comment:
I think we should format the output a little bit, what do you think of this ?
```suggestion
                    "Copy and paste the 🔗 below to your browser:\n{}?project={}",
```

Reply:
ya sounds good.

Reply:
updated